app/kubernetes/components/datatables/resource-pools-datatable/resourcePoolsDatatableController.js

```diff
@@ -41,6 +46,7 @@ angular.module('portainer.docker').controller('KubernetesResourcePoolsDatatableC
     };
 
     this.$onInit = function () {
+      this.endpoint = EndpointProvider.currentEndpoint();
```

Comment:
EndpointProvider is deprecated. https://portainer.atlassian.net/wiki/spaces/TECH/pages/45023521/Front-end+Style+Guide#More

either receive the endpoint object from the view, or just the `RestrictDefaultNamespace` value